test/test_sanity.rb

```diff
@@ -15,16 +15,16 @@ def config subconfig = nil
     end
   end
 
-  def assert_init exp_all, cfg = nil
+  def assert_init exp_max_score, cfg = nil
     root = "."
-    exp = ["./lib/cc/engine/flog.rb", "./test/test_sanity.rb"]
+    exp = ["./test/test_sanity.rb", "./lib/cc/engine/flog.rb"]
```

Comment:
Probably need to explicitly sort this and the actual result below. Filesystem ordering on linux vs osx is annoying.